pulser-core/pulser/sequence/sequence.py

```diff
@@ -205,21 +230,46 @@ def declared_variables(self) -> dict[str, Variable]:
     @property
     def available_channels(self) -> dict[str, Channel]:
```

Comment:
I think this method can be refactored now that we have `_in_ising` and `_in_xy`

Reply:
Changes are coming ;)